docs/website/docs/plus/getting-started/advanced_tutorial.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Advanced Project tutorial
+description: Using the dlt+ cli commands to create and manage dlt+ Project
+keywords: [command line interface, cli, dlt init, dlt+, project]
+---
+
+import Link from '../../_plus_admonition.md';
+
+<Link/>
+
+This tutorial will introduce you to advanced dlt project features. You will learn how to:
+
+* Understand how dlt+ creates implicit dlt entites if they are not defined and how to prevent this
+* Learn how to use dlt+ datasets to restrict access to certain datasets and destinations
+* Learn how to interact with the dlt+ project context and entities from code
+* Create a packaged dlt+ project which you can distribute to other stakeholders in your company
+
+## Prerequisites
+
+To follow this tutorial, make sure:
+
+- dlt+ is set up according to the [installation guide](./installation.md)
+- you're familiar with the [core concepts of dlt](../../reference/explainers/how-dlt-works.md)
+- you have completed the [basic project tutorial](./tutorial.md)
+
+## Implicit entities
+
+By default, dlt+ will create implicit entities when they are requested by the user or the executed code. Let's create an example project again with `dlt project init arrow duckdb --package my_dlt_project`.
+
+```yaml
+# your sources are the data sources you want to load from
+sources:
+  arrow:
+    type: sources.arrow.source
+
+# your destinations are the databases where your data will be saved
+destinations:
+  duckdb:
+    type: duckdb
+
+# your pipelines orchestrate data loading actions
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+
+If you run the pipeline with `dlt pipeline my_pipeline run`, dlt+ will use the source and the destination referenced by the pipeline configuration. Datasets are not defined in this example, so dlt+ will create an implicit dataset with the name as defined in the pipeline configuration. For this example, you do not even need the duckdb destination defined, as this would also be implicitely created for you. 
+
+If you want to prevent dlt+ from creating implicit entities, you can set the `allow_undefined_entities` option to `false` in the project configuration by adding the following to your `dlt.yml` file:
+
+```yaml
+project:
+  allow_undefined_entities: false
+```
+
+If you try to run the pipeline "my_pipeline" now, you will receive an error that the dataset `my_pipeline_dataset` does not exist. To fix this, you can create the dataset explicitly in the `dlt.yml` file:
+
+```yaml
+datasets:
+  my_pipeline_dataset:
+    destination:
+        - duckdb
+```
+
+Note that the destination property is an array of destinations on which this dataset may be materialized. More on that in the next section.
+
+
+## Managing Datasets and Destinations
```

Comment:
I think Violetta mentioned this to you already but I don't think this section (managing datasets and destinations) or the previous one (implicit entities) belongs in this tutorial